src/Apps/Collect/Routes/Collection/Components/Header/CollectionFeaturedArtists.tsx

```diff
@@ -0,0 +1,124 @@
+import { Column, GridColumns, Text } from "@artsy/palette"
+import { EntityHeaderArtistFragmentContainer } from "Components/EntityHeaders/EntityHeaderArtist"
+import { createFragmentContainer, graphql } from "react-relay"
+import { CollectionFeaturedArtists_collection$data } from "__generated__/CollectionFeaturedArtists_collection.graphql"
+import { CollectionFeaturedArtists_artworks$data } from "__generated__/CollectionFeaturedArtists_artworks.graphql"
+import { compact, filter, take } from "lodash"
+import { ContextModule } from "@artsy/cohesion"
+
+interface CollectionFeaturedArtistsProps {
+  collection: CollectionFeaturedArtists_collection$data
+  artworks: CollectionFeaturedArtists_artworks$data
+}
+
+export const CollectionFeaturedArtists: React.FC<CollectionFeaturedArtistsProps> = ({
```

Comment:
just cut/page code out of collection header so that we could move it into the query renderer